com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/IOSBundler.java

```diff
@@ -501,11 +501,11 @@ public void bundleApplication(Project project, File bundleDir, ICanceled cancele
             // Sign any .dylib files in the Frameworks folder
             if (frameworksDir.exists()) {
                 logger.log(Level.INFO, "Signing ./Frameworks folder");
-                for (File file : frameworksDir.listFiles(File::isFile)) {
+                for (File file : frameworksDir.listFiles()) {
 
                     BundleHelper.throwIfCanceled(canceled);
 
-                    if (!file.getName().endsWith(".dylib"))
+                    if (!file.getName().endsWith(".dylib") && !file.getName().endsWith(".framework"))
```

Comment:
Nice, I didn't know you could sign the entire directory (I guess this is tested?)

Reply:
A few lines below we call codesign command for root appDir. So here is the same mechanism. 
Here is how this command looks inside XCode build process

> /usr/bin/codesign --force --sign <indentity> --timestamp\=none --preserve-metadata\=identifier,entitlements,flags <rot_folder>/TestApp/DerivedData/FamilyIsland/Build/Products/Debug-iphoneos/TestApp.app/Frameworks/SomeFramework.framework